### PR TITLE
Add tree chop minigame

### PR DESCRIPTION
**Pull request by savipauk**
- Adds quicktime event minigame skeleton
- Adds tree chop quikctime minigame using skeleton

### CODE REVIEW

**Comment by savipauk** [comment]
there are some files to clean up (sprite duplicates) - I will clean this up later so if you approve don't merge yet

**Comment by savipauk** [comment]
Cleaned up now, ready for review/merge if all good @dean-rakic @marinprusac

File: inventory/inv_scripts/inventory_slot.gd [hunk 1]
```diff
@@ -7,9 +7,11 @@ func update(slot: InventorySlot):
...
 		item_visual.visible = false
 		amount_text.visible = false
-	else:
-		item_visual.visible = true
-		item_visual.texture = slot.item.texture
-		if slot.amount > 1:
-			amount_text.visible = true
-		amount_text.text = str(slot.amount)
+		return
+	if (slot.item.name == "axe"):
```
Comment (marinprusac) [inline inventory/inv_scripts/inventory_slot.gd:11]: This makes me cry, but fine...

**Review by marinprusac — APPROVED** [review]
Nice! Good code.

**Pull request merged**